packages/restlessness-cli/rln-backend/src/models/Endpoint/index.ts

```diff
@@ -18,33 +14,24 @@ export {
   HttpMethod,
 };
 
-interface JsonEndpoint {
-  id: number,
-  route: string,
-  method: HttpMethod
-  authorizer: string
-}
+// interface JsonEndpoint {
+//   id: number,
+//   route: string,
+//   method: HttpMethod
+//   authorizer: string
+// }
 
 export default class Endpoint {
   id: number
   route: Route
   method: HttpMethod
   authorizer: Authorizer
 
-  static get endpointsJsonPath(): string {
-    return path.join(getPrjRoot(), 'endpoints.json');
-  }
-
-  static get functionsJsonPath(): string {
-    return path.join(getPrjRoot(), 'functions.json');
-  }
-
   static async getList(): Promise<Endpoint[]> {
-    const file = await fs.readFile(Endpoint.endpointsJsonPath);
-    const endpoints: JsonEndpoint[] = JSON.parse(file.toString());
+    const endpoints: JsonEndpoint[] = await JsonEndpoint.getList<JsonEndpoint>();
     return endpoints.map(endpoint => {
       const ep = new Endpoint();
-      ep.id = endpoint.id;
+      ep.id = +endpoint.id;
```

Comment:
this will be a string, I think we should treat backend IDs as strings as well

Reply:
ok, i'm going to change it